guides/aws/aws.md

```diff
@@ -75,11 +75,33 @@ kubectl get secrets juice-balancer-secret -o=jsonpath='{.data.adminPassword}' |
 
 ## Step 4. Add Ingress to expose the app to the world
 
-**WARNING:** I, as a AWS Noob, haven't yet figured out how to get it working correctly.
-The Guide below shows **how I thing it should work** but it doesn't. At least not for me. If you are a AWS Pro please please send me a message / open up an issue / pull request correcting this section.
+First, we need to create an iam policy which gives permissions to create the load balancer.
 
-AWS let's you create LoadBalancer by adding a new ingress config to you cluster.
-To set this up follow the **To deploy the ALB Ingress Controller to an Amazon EKS cluster** guide on https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html closely. This will walk you through setting up and configuring the ingress.
+```sh
+#Take note of the ARN of the Policy
+aws iam create-policy \
+--policy-name ALBIngressControllerIAMPolicy \
+--policy-document https://raw.githubusercontent.com/kubernetes-sigs/aws-alb-ingress-controller/v1.1.4/docs/examples/iam-policy.json
+```
+
+Next, we will integrate Kubernetes with AWS, allowing the Kubernetes to provision an Application load balancer on our behalf.
+
+```sh
+#Associate IAM OIDC Provider
+wget https://raw.githubusercontent.com/kubernetes-sigs/aws-alb-ingress-controller/v1.1.4/docs/examples/cluster-iam.yaml
```

Comment:
I'm getting a 404 when trying to download `https://raw.githubusercontent.com/kubernetes-sigs/aws-alb-ingress-controller/v1.1.4/docs/examples/cluster-iam.yaml`.

The folder doesn't seem to contain a `cluster-iam.yaml` file: https://github.com/kubernetes-sigs/aws-alb-ingress-controller/tree/v1.1.4/docs/examples

Reply:
Also probably best to update the eks cluster to k8s 1.14 when using all the 1.14 resources.

Reply:
Ah, I messed up. Most of the links were supposed to point to the repo version.
I'll see if i can get the declarative version out before submitting another PR, as it will invalid this.

Reply:
Okay nice 👍